lib/tower_cli/models/base.py

```diff
@@ -312,9 +312,19 @@ def _format_yaml(self, payload):
                 """Convert the payload into a YAML string with proper
                 indentation and return it.
                 """
-                payload = json.loads(json.dumps(payload))
-                return yaml.safe_dump(payload, indent=2, allow_unicode=True,
-                                      default_flow_style=False)
+                def ordered_dump(data, stream=None, Dumper=yaml.Dumper, **kws):
```

Comment:
Putting this in a function seems a little more complicated than we need. We don't have any intention of reusing it, so maybe we could just remove the function and keep the last line as `return yaml.dump(...)`?